src/components/templates/ExperimentalSpace/ExperimentalSpace.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useCallback, useEffect, useMemo } from "react";
```

Comment:
This file doesn't need reviewing. It's a space type purely for prototyping "projecting" video to a central pane. It'll be deleted.

Reply:
I guess the above comment should have been useful inside the file